api-server/src/server/boot/challenge.js

```diff
@@ -69,14 +69,18 @@ export default async function bootChallenge(app, done) {
   done();
 }
 
-const jsProjects = [
+const jsCertProjectIds = [
   'aaa48de84e1ecc7c742e1124',
   'a7f4d8f2483413a6ce226cac',
   '56533eb9ac21ba0edf2244e2',
   'aff0395860f5d3034dc0bfc9',
   'aa2e6f85cab2ab736c9a9b24'
 ];
 
+const multiFileCertProjectIds = getChallenges()
+  .filter(challenge => challenge.challengeType === 14)
+  .map(challenge => challenge.id);
```

Comment:
Is this not unnecessary now that we attached the `usesMultifileEditor` property to the meta?

Reply:
no - there may be a better method, but this is a list of the id's for the cert projects (challengeType === 14). It determines if we want to save the files in the database or not. usesMultiFileEditor would save the files for all the practice projects.